engine/src/main/java/org/terasology/engine/physics/bullet/BulletPhysics.java

```diff
@@ -241,7 +241,8 @@ public HitResult rayTrace(Vector3f from, Vector3f direction, float distance, Col
                             hitNormalWorld,
                             voxelPosition);
                 } else { //we hit something we don't understand, assume its nothing and log a warning
-                    logger.warn("Unidentified object was hit in the physics engine: {}", collisionObject.userData);
+                    logger.atWarn().addArgument(() -> collisionObject.userData).
+                            log("Unidentified object was hit in the physics engine: {}");
```

Comment:
simple member access so impact neglectable
```suggestion
                    logger.warn("Unidentified object was hit in the physics engine: {}", collisionObject.userData); //NOPMD
```